src/skins/vector/css/matrix-react-sdk/views/messages/TextualEvent.css

```diff
@@ -15,6 +15,11 @@ limitations under the License.
 */
 
 .mx_TextualEvent {
-    opacity: 0.5;
-    overflow-y: hidden;
+    font-weight: 600;
```

Comment:
ah, ok, i'd totally missed that.  I agree that improves the visual rhythm of the page.  However, I am concerned that UX-wise it's taking stuff which isn't so important to users (joins/parts/avatar-changes) which generally should fade into the background and making them seem more urgent by bolding them.

Unsure how to resolve this one... are there any alternatives that you'd consider?

(Separately, I'd very much to keep /me style actions formatted the same as the surrounding text, given they should just be read as a different type of message, unlike Slack which italicises them etc.  However, this behaviour is still intact).